File: SmashBro_8-1 v0.2/smashbor_ver0.1/Camera.cpp
#include "Camera.h"

void CCamera::setPos(int x) {
	if (x <= -700 || x >= 700);
	else {
		pos.x = x*(1280 / 940) - 640; pos.y = 0;
		realpos.x = x*(1280 / 940) - 640; realpos.y = 0;
	}
}
void CCamera::setPos(int x1, int x2) {
	pos.x = ((x1 + x2) / 2)*(1280 / 940) - 640; pos.y = 0;
	realpos.x = ((x1 + x2) / 2)*(1280 / 940) - 640; realpos.y = 0;
}
void CCamera::realsetPos(int x) {
	if (x <= -700 || x >= 700);
	else {
		realpos.x = x*(1280 / 940) - 640; realpos.y = 0;
	}
}
void CCamera::realsetPos(int x1, int x2) {
	realpos.x = ((x1 + x2) / 2)*(1280 / 940) - 640; realpos.y = 0;
}
void CCamera::add() {
	if ((realpos.x != pos.x))
		pos.x += (realpos.x - pos.x) / 20;
}

File: android/SuperMario_2/SuperMario/app/src/main/java/com/example/parkjaeha/supermario/Camera.java
package com.example.parkjaeha.supermario;

/**
 * Created by woolo_so5omoy on 2017-08-27.
 */

public class Camera {
    POINT pos=new POINT();
    POINT realpos=new POINT();
    POINT getPos(){
        return pos;
    }
    void setPos(int x){
        if(x<-1400||x>=1400);
            else{
            pos.x=x*(2560/1880)-1280;
            pos.y=0;
            realpos.x=x*(2560/1880)-1280;
            realpos.y=0;
        }
    }
    void realsetPos(int x){
        if(x<=-1400||x>=1400);
        else{
            realpos.x=x*(2560/1880)-1280;
            realpos.y=0;
        }
    }
    void add(){
        if(realpos.x!=pos.x)
            pos.x+=(realpos.x-pos.x)/20;
    }
}
